package.json

```diff
@@ -3,23 +3,22 @@
   "version": "0.0.1",
   "description": "OAS (Swagger v3) API Definition for Template API",
   "scripts": {
-    "lint": "node_modules/.bin/speccy lint -s openapi-tags -s operation-tags specification/shared-flow-testing.yaml --skip default-and-example-are-redundant",
-    "publish": "mkdir -p build && node_modules/.bin/speccy resolve specification/shared-flow-testing.yaml -i | poetry run python scripts/yaml2json.py | poetry run python scripts/set_version.py > build/shared-flow-testing.json",
-    "serve": "node_modules/.bin/speccy serve build/shared-flow-testing.json",
+    "lint": "node_modules/.bin/openapi-generator-cli validate -i specification/shared-flow-testing.yaml",
+    "publish": "./node_modules/.bin/openapi-generator-cli generate -i specification/shared-flow-testing.yaml --generator-key v3.0  -g openapi -o build && rm openapitools.json && mv build/openapi.json build/shared-flow-testing.json",
     "check-licenses": "node_modules/.bin/license-checker --failOn GPL --failOn LGPL"
   },
   "author": "NHS Digital",
   "license": "MIT",
   "homepage": "https://github.com/NHSDigital/shared-flow-testing",
   "dependencies": {
-    "newman-reporter-htmlextra": "^1.22.6",
-    "speccy": "^0.11.0"
+    "newman-reporter-htmlextra": "^1.22.6"
```

Comment:
Are the newman packages still required for this repo? For some of the others they were removed